recipes/icu/all/conandata.yml

```diff
@@ -17,6 +17,9 @@ sources:
   "68.2":
     url: "https://github.com/unicode-org/icu/releases/download/release-68-2/icu4c-68_2-src.tgz"
     sha256: "c79193dee3907a2199b8296a93b52c5cb74332c26f3d167269487680d479d625"
+  "69.1":
+    url: "https://github.com/unicode-org/icu/releases/download/release-69-1/icu4c-69_1-src.tgz"
+    sha256: "39ce83dd5d15c7539dde261733e106a391923f82caf1ce52ecaebb72d93b4579"
```

Comment:
```suggestion
    sha256: "4cba7b7acd1d3c42c44bb0c14be6637098c7faf2b330ce876bc5f3b915d09745"
```

Reply:
![image](https://user-images.githubusercontent.com/4138939/120873192-5ddb6d00-c5a1-11eb-8ff6-1406e1e89626.png)

Reply:
That's on me, I used the tag's archive instead of the release archive

Reply:
Oh I thought the release managers fudged it.